### PR TITLE
Pp um cutout

### PR DESCRIPTION
**Pull request by pp-mo**
Taking this forward from Laura's code : https://github.com/SciTools/iris/pull/1619.
Eyes on the existing code would be useful.
I've made some changes here, but the more important changes are still to come :
- add an integration test
- fix the problem with Iris loads that @lbdreyer mentions

### CODE REVIEW

File: lib/iris/experimental/um.py [hunk 1]
```diff
@@ -726,20 +725,43 @@ def cutout(ffv_src, filename, cutout_params):
 
     * ffv_src (:class:`FieldsFileVariant`)
         A source FieldFileVariant to subset.
-    * filename (string)
-        The name of the file containing the subset of the source UM
-        FieldsFile variant.
+    * filepath (string)
```
Comment (pelson) [inline lib/iris/experimental/um.py]: @pp-mo: http://en.wikipedia.org/wiki/Filename

Comment (pp-mo) [inline lib/iris/experimental/um.py]: I'm not saying 'filename' is wrong, I just find 'filepath' clearer.
I try to always make this distinction as it's been so helpful when messing with paths, especially in our hopefully OS-neutral context.  So I think it has become automatic.
Actually, I should edit the description as well...

Comment (pp-mo) [inline lib/iris/experimental/um.py]: Ok, it doesn't match the usage in FieldsFileVariant itself, which is confusing, so I'll change it back...

**Comment by pp-mo** [comment]
I think this does it all now.
Review, anyone (@pelson @rhattersley) ?

**Comment by pp-mo** [comment]
Close this now.
We don't get Travis testing in this repo.
I'll target this against master instead....

**Pull request closed without merging**